docs/user/dql/basics.rst

```diff
@@ -155,6 +155,46 @@ Result set:
 |  Nanette|   Bates|
 +---------+--------+
 
+One can also provide meta-field name(s) in ``SELECT`` clause to retrieve reserved-fields (beginning with underscore) from OpenSearch documents.
+
+SQL query::
+
+	POST /_plugins/_sql
+	{
+	  "query" : "SELECT firstname, lastname, _id, _index, _sort FROM accounts"
+	}
+
```

Comment:
Updated - although returning `_id` doesn't work because we cannot set the id in doctests.

Reply:
I think it is set for `books` table

Reply:
Unfortunately, no.  
The only one that has the _id set is [wildcard](https://github.com/opensearch-project/sql/blob/main/doctest/test_data/wildcard.json)...but it actually doesn't get imported because it's not using the _bulk endpoint (in fact, the data is read-in as a separate row... `{"index":{"_id":"0"}}`).